tests/test_wandb_integration.py

```
@@ -0,0 +1,698 @@
+from stormlog._wandb.tracking import _TIMELINE_MAX_POINTS, sample_timeline_rows
+from stormlog.session import create_session_summary
+from stormlog.wandb_integration import (
+    ensure_wandb_available,
+    export_diagnose_bundle_to_wandb,
+    export_tracking_run_to_wandb,
+    wandb_config_from_namespace,
+)
+
+
+class _FakeArtifact:
+    def __init__(self, name: str, type: str) -> None:
```

Ruff (0.15.10)

[error] Function argument `type` is shadowing a Python builtin

(A002)